### PR TITLE
Add orka-vm-tools v2.2

### PR DESCRIPTION
**Pull request by ispasov**
A cask for Orka VM Tools.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by p-linnane** [comment]
@ispasov Please try to add a `zap` stanza. Information one how to create one can be found [here](https://docs.brew.sh/Cask-Cookbook#stanza-zap).

**Comment by ispasov** [comment]
hey @p-linnane quick question about the zap stanza.
According to the documentation it may remove files from the user's library folder or shared resources.
The VM tools create neither. 
The files that are created are inside:
- `/Library/LaunchDaemons`
- `/Applications`
and are automatically deleted by the uninstaller.

Do I need zap for that?

Also, I tried the `createzap` tool and this is the output:

```
❯ brew createzap orka-vm-tools
zap trash: "~/orka-vm-tools.tar.gz"
```

So to sum up:
1. Do I need zap stanza for the files created by the installer?
2. Should I trust the createzap tool?

**Comment by p-linnane** [comment]
@ispasov The intent of a `zap` stanza is to remove anything left over after uninstall is completed, so no those don't need to be included.

You also do not need to include the file that `createzap` lists. I believe that file is showing up because it's in your `/Users/` folder and contains `"orka"`. The cask itself is now pulling down that file or interacting with it.

This should be good to merge after as is once a maintainer reviews and approves it. Thanks!

**Comment by miccal** [comment]
Thank you @ispasov.

**Review by miccal — APPROVED** [review]
